recipes/aiofiles/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "aiofiles" %}
+{% set version = "0.3.2" %}
+{% set sha256 = "852a493a877b73e11823bfd4e8e5ef2610d70d12c9eaed961bcd9124d8de8c10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py<33]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - asyncio  # [py<34]
+    - singledispatch  # [py<34]
+
+test:
+  imports:
+    - aiofiles
+
+about:
+  home: https://github.com/Tinche/aiofiles
+  summary: "File support for asyncio"
+  license: Apache 2.0
```

Comment:
You could add `license_family: Apache` too, while you're editing stuff for the other pending changes.